pom.xml

```diff
@@ -7,7 +7,7 @@
 
     <groupId>io.dropwizard.modules</groupId>
     <artifactId>dropwizard-protobuf</artifactId>
-    <version>0.7.1-4-SNAPSHOT</version>
+    <version>0.7.1-4-protobuf${protobuf.version}-SNAPSHOT</version>
```

Comment:
I don't want to include the protobuf version in the package version as that doesn't follow the versioning convention as defined in https://groups.google.com/d/msg/dropwizard-dev/nY23U4996_k/rKP_e05sIRAJ